packages/plugin-selenium-driver/src/plugin/index.ts

```diff
@@ -421,358 +429,301 @@ export class SeleniumPlugin implements IBrowserProxyPlugin {
+        return selector.isDisplayed();
     }
 
     public async moveToObject(applicant: string, xpath: string, x: number, y: number) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            const elem = await client.$(xpath);
-            await elem.scrollIntoView();
-            return elem.moveTo(x, y);
-        }
+        const elem = await client.$(xpath);
+        await elem.scrollIntoView();
+        return elem.moveTo(x, y);
     }
 
     public async execute(applicant: string, fn: any, args: Array<any>) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return client.execute(fn, ...args);
-        }
+        return client.execute(fn, ...args);
     }
 
     public async executeAsync(applicant: string, fn: any, args: Array<any>) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return client.executeAsync(fn, ...args);
-        }
+        return client.executeAsync(fn, ...args);
     }
 
     public async getTitle(applicant: string) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return client.getTitle();
-        }
+        return client.getTitle();
     }
 
     public async clearElement(applicant: string, xpath: string) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return (await client.$(xpath)).clearValue();
-        }
+        const selector = await client.$(xpath);
+        return selector.clearValue();
     }
 
     public async keys(applicant: string, value: any) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return client.keys(value);
-        }
+        return client.keys(value);
     }
 
     public async elementIdText(applicant: string, elementId: string) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            return client.getElementText(elementId);
-        }
+        return client.getElementText(elementId);
     }
 
     public async elements(applicant: string, xpath: string) {
         await this.createClient(applicant);
         const client = this.getBrowserClient(applicant);
 
-        if (client) {
-            const elements = await client.findElements('xpath', xpath) as unknown;
-            const a = (elements as Array<Record<string, string>>).map((o) => {
-                const keys = Object.keys(o);
-                return { 'ELEMENT': o[keys[0]] };
-            });
-
-            return a;
-        }
+        const elements = await client.findElements('xpath', xpath) as unknown;
```

Comment:
Is it needed to cast `uknowkn` and then back to `Array<Record<string, string>>` why not to do it right in place?